custom/icds_reports/templates/icds_reports/icds_app/reassignment-message.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-info" ng-if="selectedLocation.deprecated_at">
+  {$ selectedLocation.name $} was inactive in the selected month. It was replaced by {$ selectedLocation.deprecated_to.join(' & ') $} on {$ selectedLocation.deprecated_at $}.
+</div>
+<div class="alert alert-info" ng-if="selectedLocation.deprecates_at">
+  {$ selectedLocation.name $} was inactive in the selected month. It replaced {$ selectedLocation.deprecates.join(' & ') $} on {$ selectedLocation.deprecates_at $}.
```

Comment:
just confirming that here `deprecates` and `deprecated_to` are location names and not the id? I guess you already mapped that somewhere but could not point to it, so just confirming.

Reply:
Yeah, thats what the util changes do (the function `get_deprecation_info` is for finding the names of the relevant locations)